bin/scripts/fix-data/recheck_merges.js

```diff
@@ -0,0 +1,89 @@
+/**
+ *  Description: This script is used to recheck merges and update drill data with new uid
+ *  Server: countly
+ *  Path: $(countly dir)/bin/scripts/fix-data
+ *  Command: node recheck_merges.js
+ */
+
+
+var asyncjs = require("async");
+const pluginManager = require('../../../plugins/pluginManager.js');
+const dataviews = require('../../../plugins/drill/api/parts/data/dataviews.js');
+
+Promise.all([pluginManager.dbConnection("countly"), pluginManager.dbConnection("countly_drill")]).then(async function([countlyDb, drillDb]) {
+    console.log("Connected to databases.");
+    //get all apps
+    countlyDb.collection("apps").find().toArray(function(err, apps) {
+        if (err) {
+            return close(err);
+        }
+        //get all drill collections
+        drillDb.collections(function(err, collections) {
+            if (err) {
+                return close(err);
+            }
+            //for each app
+            asyncjs.eachOf(apps, function(app, index, done) {
```

Comment:
We need this script to not be the fastest, but slowly work not overwhelming system.
Based on documentation it means that it will run in parallel. We want it to be in series. New app should be processed only when first one is finished:

http://caolan.github.io/async/v3/docs.html#eachOf - states it works like each
http://caolan.github.io/async/v3/docs.html#each states it works in paralel